### DISCUSSION TITLE
Do you see Comet for $5 or for free in AppCenter?

### DISCUSSION THREAD

**Discussion by aral** [elementary/appcenter#1798] [Q&A] [answered]
Hey folks,

Can you let me know if you see Comet on AppCenter for $5 or for free? 

I tried buying it earlier today when it had the wrong Stripe key and I don’t know if that’s what did it but I can only see the Free button (so I can’t test the payment functionality).

Wondering if it’s just on my machine or in general.

Thanks!

**Comment by aral** [comment]
PS. If I press the Free button, I see the $5 button flash briefly so I’m thinking it’s some odd state my AppCenter has gotten itself into.

**Comment by meisenzahl** [comment, marked as answer]
![Bildschirmfoto von 2021-12-18 09 22 37](https://user-images.githubusercontent.com/10796736/146634714-8eb522c5-de16-4682-832a-545e02e5917e.png)

**Comment by aral** [comment]
@meisenzahl Thanks! :)

I also saw it featured on the day it went up but it seems to have popped off (perhaps just for me).

Need to track down where such things are cached and try clearing it, I guess :)

**Reply by cassidyjames** [reply]
If you've previously purchased it, it will show as free in the future! You can check `io.elementary.appcenter.settings paid-apps` in Dconf Editor to see if it's marked as previously having been paid for.
